app/src/androidTest/java/org/mozilla/tv/firefox/ui/robots/NavigationOverlayRobot.kt

```diff
@@ -97,6 +102,23 @@ class NavigationOverlayRobot {
     fun disableSessionIdling(activityTestRule: MainActivityTestRule) { activityTestRule.loadingIdlingResource.ignoreLoading = true }
     fun enableSessionIdling(activityTestRule: MainActivityTestRule) { activityTestRule.loadingIdlingResource.ignoreLoading = false }
 
+    // Focus the pin button using keypresses, starting from urlbar assuming back is disabled, forward is enabled
+    fun navigateToPinButton() {
```

Comment:
Perhaps we should change the title of this method for clarity, since it assumes it's always starting from URLbar.
navigateToPinBtnWhenURLBarSelected() would give less room for confusion.  Either, we could create a method that taps left or right key x times, and comment in the main test script that x taps to right would give navigatedToPinBtn selected.